actpool/actionstore.go

```diff
@@ -56,7 +55,6 @@ type (
 	}
 	actionStoreConfig struct {
 		Datadir string `yaml:"datadir"` // Data directory containing the currently executable blobs
-		Datacap uint64 `yaml:"datacap"` // Soft-cap of database storage (hard cap is larger due to overhead)
```

Comment:
for v2.1.0 release, we added this config into `config.yaml`. AFAI recall, `datacap` is not added in the yaml file, but please make sure, so that existing full-node can start normally with existing yaml file

Reply:
yes, it has not been written in the config